frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-actions.feature

```diff
@@ -86,14 +86,14 @@ Feature: Perform the actions on Pipelines page
                   | pipelines-ccc |
 
 
-        @to-do
+        @regression
         Scenario Outline: Add the task by editing the pipeline: P-06-TC08
             Given pipeline "<pipeline_name>" is present on Pipelines page
              When user selects "Edit Pipeline" option from kebab menu of "<pipeline_name>"
              When user clicks on Add task in parallel
```

Comment:
just noticed there are 2 When lines, can you change the second one to And to keep it consistent?

Reply:
changes done!